libs/shared/src/canvas/runtime/node.ts

```diff
@@ -21,9 +21,9 @@ export const startCanvasNode = async (config: { PEER_ID?: string }) => {
   const listen =
     process.env.FEDERATION_LISTEN_ADDRESS ?? '/ip4/127.0.0.1/tcp/8090/ws';
```

Comment:
We could move some of these process.envs into `config` (later?) if we want to consolidate config params. It's done inconsistently elsewhere so I haven't changed anything yet.

Reply:
Is there a reason any of this is in `libs/shared`? It seems to be a mistake. `libs/shared` is meant to be for code used in both client and server, but unless I am mistaken, `startCanvasNode` is a server-side-only operation. There is no `config` for `libs/shared` because dotenv is not supported on the client, and we have a separate method of loading configuration variables for the client using the build process.

All new server-side environment variables should be in a `config` object. Any that aren't is only because they haven't been transferred yet (or it is a mistake that they weren't added to a `config` object).

Reply:
From discussion in Slack, this happened because of a request to move Canvas code to libs/shared and will be fixed in #9718.